modules/installation-and-upgrade/pages/container-management/mgradm-yaml-custom-configuration.adoc

```diff
@@ -9,37 +9,46 @@ You have the option to create a custom [filename]``mgradm.yaml`` file, which the
 For security, **using command line parameters to specify passwords should be avoided**: use a configuration file with proper permissions instead.
 ====
 
-.Procedure: Deploying an Uyuni container with Podman using a custom configuration file
-. Prepare an mgradm.yaml file similar to the following example:
+.Procedure: Deploying an {productname} container with Podman using a custom configuration file
+. Prepare a configuration file named [filename]``mgradm.yaml``mgradm.yaml`` similar to the following example:
 +
+
 [source, yaml]
 ----
-# Database password
+# Database password. Randomly generated by default
 db:
   password: MySuperSecretDBPass
-# Certificate password
-cert:
-  password: MySuperSecretCAPass
-# SSL password
+
+# Password for the CA certificate
 ssl:
-  password: MySuperSecretSllPassword
+  password: MySuperSecretSSLPassword
 
-# If you have these you can provide them to setup your organization credentials
+# Your SUSE Customer Center credentials
 scc: 
   user: ccUsername
   password: ccPassword
 
-# Administrator email address
-email: email@example.com
+# Organization name
+organization: YourOrganization
+
+# Email address sending the notifications
+emailFrom: notifications@example.com
 
+# Administrators account details
+admin:
+  password: MySuperSecretAdminPass
+  login: LoginName
+  firstName: Admin
+  lastName: Admin
+  email: email@example.com
 ----
```

Comment:
@cbosdo can you confirm this? I took a look at what we do in https://github.com/uyuni-project/sumaform/blob/master/salt/server_containerized/mgradm.yaml and adapted this example.

Reply:
The example looks good to me. I would love the docs people to have a look at the wording too

Reply:
Thank you for confirming this, Cedric!